src/App.tsx

```diff
@@ -9,42 +9,59 @@ import { createTheme, ThemeProvider } from "@mui/material/styles";
 
 const theme = createTheme();
 
+type USER = {
```

Comment:
型名はアッパーキャメルケースが一般的ですね！
```suggestion
type User = {
```

Reply:
修正します！